test/java/seedu/addressbook/storage/StorageFileTest.java

```diff
@@ -25,7 +25,7 @@
 import static seedu.addressbook.util.TestUtil.assertFileDoesNotExist;
 
 public class StorageFileTest {
-    private static final String TEST_DATA_FOLDER = "test/data/StorageFileTest";
+    private static final String TEST_DATA_FOLDER = "/Users/jamespang/Desktop/school/y2s2/CS2103/Project/addressbook-level2/test/data/StorageFileTest";
```

Comment:
Why add a path that is only specific to your own computer? This will not work on other computers.